lib/src/constants/enums.dart

```diff
@@ -6,7 +6,26 @@
 /// Provide some special picker types to integrate
 /// un-common pick pattern.
 /// 提供一些特殊的选择器类型以整合非常规的选择行为。
-enum SpecialPickerType { wechatMoment }
+enum SpecialPickerType { 
+  /// WeChat Moments mode.
+  /// 微信朋友圈模式
+  ///
+  /// The user can only select *one video* or *multiple images* at the same time,
+  /// and those two asset types cannot be selected at the same time.
+  /// 用户只可以选择 **一个视频** 或 **多个图片**，并且两种类型互斥。
+  wechatMoment,
+  /// Disable preview of assets.
+  /// 禁用资产预览
```

Comment:
Ah you dropped my chinese translations haha.

```suggestion
  /// 禁用资源预览
```